src/app/modules/workspace/workspace.controller.ts

```diff
@@ -98,24 +94,37 @@ export const createWorkspace = [
  *
  * @returns 200 with updated workspace
  * @returns 404 when workspace cannot be found
- * @returns 422 when user of given id cannnot be found in the database
  * @returns 500 when database error occurs
  */
-const handleUpdateWorkspaceTitle: ControllerHandler<
-  unknown,
-  any | ErrorDto,
+export const handleUpdateWorkspaceTitle: ControllerHandler<
+  { workspaceId: string },
+  WorkspaceDto | ErrorDto,
   { title: string }
 > = async (req, res) => {
+  const { workspaceId } = req.params
   const { title } = req.body
-  return WorkspaceService.updateWorkspaceTitle('', title)
+  const userId = (req.session as AuthedSessionData).user._id
+
+  return WorkspaceService.updateWorkspaceTitle(workspaceId, title, userId)
```

Comment:
should we also return an appropriate error instead of `res.json(null)` when the user does not own a workspace with `workspaceId`?

Reply:
Yep! Updated to return an error instead